build/setMavenVersion.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+
+# This script will check $TRAVIS_TAG to see if we need to run maven to
+# set the artifact version #'s.
+
+if [[ -n "${TRAVIS_TAG}" ]]; then
+    printf "\n>>>>> Setting artifact version #'s to: %s\n" ${TRAVIS_TAG:1}
+    mvn versions:set -DnewVersion=${TRAVIS_TAG:1} -DgenerateBackupPoms=false
```

Comment:
I used ${TRAVIS_TAG:1} since we name our tags `vx.x.x`, do I still need to update this?

Reply:
ohhhh, right... yes you'd want to use a version # that omits the "v", so disregard my comment.
I think this is a good reason why I don't use the "v" in tags, so that the tag value and the maven version # are the same :)